### PR TITLE
[BFD-750] Create hospice unit tests v2

### PR DESCRIPTION
**Pull request by mackec1**
### Change Details

Add unit tests for Hospice Claims V2

### Acceptance Validation

Sufficient tests to support Sample A Hospice Claims

### Feedback Requested

Missing test coverage for Hospice claims.
Adheres to V2 unit test(s) standards.

### External References

- [BFD-750](https://jira.cms.gov/browse/BFD-750)

### Security Implications

N/A

- [ ] new software dependencies

- [ ] altered security controls

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Review by cbrunefearless — APPROVED** [review]
lgtm

File: apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/HospiceClaimTransformerV2Test.java [hunk 1]
```diff
@@ -1,74 +1,789 @@
...
+    eob = HospiceClaimTransformerV2.transform(new MetricRegistry(), claim, includeTaxNumber);
+  }
 
-    return claim;
+  /**
+   * Serializes the EOB and prints to the command line
+   *
+   * @throws FHIRException
+   */
+  // @Ignore
```
Comment (rtravitz) [inline apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/HospiceClaimTransformerV2Test.java:76]: Did you want this commented out? Generates a lot of output.

**Review by rtravitz — APPROVED** [review]
Ready to go other than those intermittent failures.

**Review by cbrunefearless — APPROVED** [review]
lgtm

**Pull request merged**